src/main/java/com/bandwidth/sdk/BandwidthClient.java

```diff
@@ -86,8 +114,16 @@ public synchronized static BandwidthClient getInstance() {
      * @param apiSecret the user API secret.
      * @param apiEndpoint the API Endpoint.
      * @param apiVersion the API version.
+     * @param maxTotal the API Endpoint.
+     * @param defaultMaxPerRoute the API version.
      */
-    protected BandwidthClient(final String userId, final String apiToken, final String apiSecret, final String apiEndpoint, final String apiVersion){
```

Comment:
If you decide to go for this approach, overload this method to keep the backward compatibility